lib/src/slider/components/custom_slider_thumb_shape.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+class CustomSliderThumbShape extends SliderComponentShape {
+  final double thumbRadius;
+
+  CustomSliderThumbShape({
+    required this.thumbRadius,
+  });
```

Comment:
While we are maintaining the practice of keeping the constructor above the declaration, please do it here as well...

Reply:
ok done...making a commit soon